addons/mail/static/src/core/web/message_patch.js

```diff
@@ -65,6 +65,7 @@ patch(Message.prototype, {
         }
     },
 
+    /** @deprecated */
     async onClickMessageForward() {
         const email_from = this.message.author?.email
             ? this.message.author.email
```

Comment:
I would not keep the deprecated code there, instead I would map the inner-code to the newer code.
That way the deprecated code has minimal footprint and very easy to remove.